resource/ui/ts/models/log.ts

```diff
@@ -16,35 +16,43 @@ module Models {
      */
     export module Log {
         import promise = _mithril.MithrilPromise;
+        import property = _mithril.MithrilProperty;
 
         export interface LogResponseSet {
             d: Proto.LogEntry[]
         }
 
         export class Entries {
-            startTime = Utils.chainProp(this, <number>null);
-            endTime = Utils.chainProp(this, <number>null);
-            max = Utils.chainProp(this, <number>null);
-            level = Utils.chainProp(this, <string>null);
+            startTime = Utils.chainProp(this, <property<number>>null);
+            endTime = Utils.chainProp(this, <property<number>>null);
+            max = Utils.chainProp(this, <property<number>>null);
+            level = Utils.chainProp(this, <property<string>>null);
 
             private _url(): string {
                 var url = "/_status/local/log";
-                if (this.level() != null) {
-                    url += "/" + this.level();
+                if ((this.level() != null) && (this.level()() != null)) {
```

Comment:
Alright, the Select component shouldn't accept a "selected" property.  It should accept a list of values, the currently selected value (as a string), and a callback.

The callback itself could be a simple property - in the log view, though, it will be something more complicated since it also needs to kick off a request.